translations/strings.yaml

```diff
@@ -337,23 +337,55 @@ telemetryPolicyView:
   description: We strive to provide you with choices and collect only the data we need to improve Mozilla VPN. Sharing this data with Mozilla is optional.
   question: Allow Mozilla VPN to send technical, interaction, and campaign and referral data to Mozilla?
 
+genericError:
+  unexpected: Unexpected error
 
+# Connection Info Screen
 connectionInfo:
   labelBitps:
-    value: bit/s
+    value: bits
```

Comment:
This applies to VPN as well: https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html

You need new IDs for this type of changes.